pycqed/tests/test_tomo_v2.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import pycqed as pq
+import os
+import numpy as np
+from pycqed.analysis import measurement_analysis as ma
+from pycqed.analysis_v2 import tomography_execute as tomography_execute
+import qutip as qt
+
+ma.a_tools.datadir = os.path.join(pq.__path__[0], 'tests', 'test_data')
+
+
+class Test_tomography_execute(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.datadir = os.path.join(pq.__path__[0], 'tests', 'test_data')
+        ma.a_tools.datadir = self.datadir
+
+
+    def test_tomo_analysis_cardinal_state(self):
+      
+      #The dataset corresponds to the 00 cardinal state.
+      tomo_object = tomography_execute.TomographyExecute(timestamp='20161124_162604',tomography_type = "MLE")
```

Comment:
Good example in the test! 

I'd argue that this should *not* use an "execute" object. That functionality should be part of the measurement analysis class. The only reason that it works like this is that a lot of hacking was done on the original measurement_analysis.